python/cudf/cudf/tests/test_parquet.py

```diff
@@ -298,6 +302,30 @@ def test_parquet_read_row_group(tmpdir, pdf, row_group_size):
     assert_eq(pdf.reset_index(drop=True), gdf, check_categorical=False)
 
 
+@pytest.mark.parametrize("row_group_size", [5, 10])
+def test_parquet_read_row_group_split(tmpdir, pdf, row_group_size):
+    fname = tmpdir.join("row_group.parquet")
+    pdf.to_parquet(fname, compression="gzip", row_group_size=row_group_size)
+
+    num_rows, row_groups, col_names = cudf.io.read_parquet_metadata(fname)
+
+    gdf = cudf.read_parquet(
+        [fname, fname],
+        row_group_list=[
+            list(range(row_groups // 2)),
+            list(range(row_groups // 2, row_groups)),
+        ],
+    )
+    gdf = gdf.reset_index(drop=True)
+
+    if "col_category" in pdf.columns:
+        pdf = pdf.drop(columns=["col_category"])
+    if "col_category" in gdf.columns:
+        gdf = gdf.drop("col_category")
+
+    assert_eq(pdf.reset_index(drop=True), gdf, check_categorical=False)
```

Comment:
Do you need to drop the categorical column **and** use `check_categorical=False` here?

Reply:
Apparently not, thanks! Modified all instances where this was done.